README.md

```diff
@@ -257,6 +257,9 @@ A selection of major game studios, publishers, etc. using GitHub:
 
 * [Open Meridian](https://github.com/OpenMeridian/Meridian59) - The first 3D MMORPG, released in 1996 and open sourced in 2012. Forked in 2013, Actively developed. [Play it now!](http://openmeridian.org)
 * [Meridian 59](https://github.com/Meridian59/Meridian59) - The first 3D MMORPG, released in 1996 and open sourced in 2012. The original codebase for Meridian 59, less frequently updated. [Play it now!](http://www.meridian59.com)
+* [The Mana World](https://github.com/themanaworld) - are a free 2D open source MMORPG, released in 2004.
+* [PlaneShift](https://github.com/baoboa/planeshift) - PlaneShift is a cross-platform 3D Online Multiplayer Role Playing Game in a fantasy setting, released in 2004.[Play it now!](http://www.planeshift.it/)
```

Comment:
I think the PlaneShift repo is an outdated copy of the [original on SourceForge](http://sourceforge.net/projects/planeshift/). Can you remove this entry?